# single cell RNA-Seq

Min Zhang
v0.1 
October 26, 2017

## Literature:

#### DropSeq:
Macosko et al Cell 2015, http://dx.doi.org/10.1016/j.cell.2015.05.002   
Shekhar et al Cell 2016, http://dx.doi.org/10.1016/j.cell.2016.07.054   

#### Fluidigm C1:
Shalek et al Nature 2014 (Regev lab),  doi:10.1038/nature13437   

#### Perturb-Seq: scRNA-Seq + CRISPR pool
Adamson et al Cell 2016, http://dx.doi.org/10.1016/j.cell.2016.11.048  
Dixit et al Cell 2016 (Regev lab), http://dx.doi.org/10.1016/j.cell.2016.11.038  

#### 10xgenomics: 


#### SmartSeq: basic for single cell RNA amplification
Picelli (Rickard Sandberg lab) et al Nature Protocol 2014, doi:10.1038/nprot.2014.006  

## People:
Aviv Regev, https://www.broadinstitute.org/regev-lab  
Rickard Sandberg, http://sandberg.cmb.ki.se/  
Ido Amit, http://www.weizmann.ac.il/immunology/AmitLab/front  
Amos Tanay, http://compgenomics.weizmann.ac.il/tanay/  

## Tools:
Monocle: pseudotime analysis, http://cole-trapnell-lab.github.io/monocle-release/  
Seurat: dropseq Macosko et al paper pipeline, good enough for basic analysis, http://satijalab.org/seurat/  
Shekhar et al paper pipelie (variation of Seurat), https://github.com/broadinstitute/BipolarCell2016  
Kallisto: pseudo alignment, super fast, https://pachterlab.github.io/kallisto/  
McCarroll Lab: dropseq data processing pipeline (Drop-seq_tools, use [STAR](https://github.com/alexdobin/STAR) and [Picard](http://broadinstitute.github.io/picard/)), wet lab protocol, http://mccarrolllab.com/dropseq/  

## GEO datasets:
